src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -158,6 +158,21 @@ function updateStatsCheckBox() {
     $('input[id=statsShownCheckBox]').val($('#show-stats-checkbox').is(':checked'));
 }
 
+function printView() {
+    // Fix to hide the filter placeholder when it is empty.
+    if ($('#results-search-box').val()) {
+        $('#results-search-box').parent().parent().removeClass('hide-for-print');
+    } else {
+        $('#results-search-box').parent().parent().addClass('hide-for-print');
```

Comment:
Using multiple `parent()` is not too good an approach here, as it may cause bugs later if the DOM is changed by other developers at a later point in time. Adding an id to the containing `div` element and using a selector with that id directly should work better.